pom.xml

```diff
@@ -12,8 +12,35 @@
     <url>https://github.com/sonyxperiadev/gerrit-events/</url>
 
     <properties>
+        <project.build.outputEncoding>UTF-8</project.build.outputEncoding>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <powermock.version>2.0.0</powermock.version>
+        <maven.compiler.release>11</maven.compiler.release>
```

Comment:
Just as an FYI that Jenkins nowadays require minimum jdk 17, so if we want to, and it makes life easier, we can go for that.

Reply:
Well then I go to 17 instead.